stubs/third_party_taint/server_side_template_injection_sinks.pysa

```diff
@@ -0,0 +1,29 @@
+# SSTI
+def jinja2.environment.Template.__new__(cls, source: TaintSink[ServerSideTemplateInjection]): ...
+def jinja2.environment.Environment.from_string(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+def jinja2.environment.Environment.compile_expression(self, source: TaintSink[ServerSideTemplateInjection, ViaTypeOf[self]]): ...
+
+#django.template
+def django.template.engine.Engine.from_string(source: TaintSink[ServerSideTemplateInjection]): ...
+def django.template.base.Template.__init__(source: TaintSink[ServerSideTemplateInjection]): ...
+
+#mako
```

Comment:
Ahh, sorry, I realize it might have been ambiguous when I said "everything from here and below" in my last comment. I meant everything on line 10 and beyond had to move to `third_party_taint`. The first 9 lines should actually stay where they were before, because they correspond to stubs and modules we have internally.

Reply:
@gbleaney sorry for not getting that right. Also, would you want both files to be named the same i.e `server_side_template_injection_sinks.pysa`?

Reply:
Yup, same naming is fine

Reply:
done